### PR TITLE
Milestone3:Module1

### PR DESCRIPTION
**Pull request by derrick1451**
### Portfolio Mobile Menu
In this milestone 

- Updated index.html

- Created script.js

- Edited styles.css

- Added popup functionality

- Once clicked on the link ,the window goes to the selected section

### CODE REVIEW

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌

Hi @derrick1451, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesn't have commits from both partners in your group. As you told me that your partner is not responding and student success asked you to submit alone. Kindly add a note with a screenshot for proof of the same.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 1]
```diff
@@ -8,21 +8,30 @@
...
+                      </a> 
+                     
+                    <div>
+                        <img class="loose" src="images/Enabled.png" alt="enable-image">
+                    </div>
+                        <ul class="pop-up">
+                            <li><a href="#portfolio"class="nav-links" >Portfolio</a></li>
+                            <li><a href="#about"class="nav-links" >About</a></li>
+                            <li><a href="#contact" class="nav-links" >Contact</a></li>
+                          </ul>
```
Comment (danmainah) [inline index.html]: - [x] Kindly check your design on the popup menu, the background should be blurred but not entirely blue as in your case check the comparison below;

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
image | image

File: index.html [hunk 2]
```diff
@@ -8,21 +8,30 @@
     <link rel="stylesheet" href="main.css">
 </head>
 <body>
-    <div class="wrapper">
             <header>
                 <nav class="nav-bar">
```
Comment (danmainah) [inline index.html]: - [ ] [optional] Kindly check on your mobile design it doesn't match with the design given;

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
image |  image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @derrick1451 and @Piercel2022 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used
✔️  Good PR and Description


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Kandy-Peter — APPROVED** [review]
Hi **@team** ,

# APPROVED 🟢 


Great implementation of all the requirements in this milestone. 👏
Your web app looks really fantastic. 🎉 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

#### CONGRATULATIONS! 🎉 

##  Optional suggestion

# CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**